src/js/page/main-controller.js

```diff
@@ -58,64 +68,66 @@ export default class MainController {
+            this._onUpdateFound(registration),
+          );
+        });
     }
 
     // tell the user about the latest update
-    storage.get('last-seen-version').then(lastSeenVersion => {
-      if (lastSeenVersion) {
-        this._changelogUi.showLogFrom(lastSeenVersion);
-      }
+    storage.get('last-seen-version').then((lastSeenVersion) => {
+      if (lastSeenVersion) this._changelogUi.showLogFrom(lastSeenVersion);
       storage.set('last-seen-version', self.version);
     });
 
     domReady.then(() => {
       this._container = document.querySelector('.app-output');
+      const actionContainer = this._container.querySelector(
+        '.action-button-container',
+      );
+      const minorActionContainer = this._container.querySelector(
+        '.minor-action-container',
+      );
+      const toolbarElement = this._container.querySelector('.toolbar');
+      const outputElement = this._container.querySelector('.output');
+      const menuExtraElement = this._container.querySelector('.menu-extra');
 
       // elements for intro anim
       this._mainUi = new MainUi(
-        document.querySelector('.toolbar'),
-        document.querySelector('.action-button-container'),
+        toolbarElement,
+        actionContainer,
         this._outputUi.container,
-        this._settingsUi.container
+        this._settingsUi.container,
       );
 
-      const actionContainer = document.querySelector('.action-button-container');
-      const minorActionContainer = document.querySelector('.minor-action-container');
-
-      minorActionContainer.appendChild(this._bgFillUi.container);
-
-      minorActionContainer.appendChild(this._copyButtonUi.container);
-      
-      actionContainer.appendChild(this._downloadButtonUi.container);
-
-      document.querySelector('.output').appendChild(this._outputUi.container);
-      this._container.appendChild(this._toastsUi.container);
-      this._container.appendChild(this._dropUi.container);
-      document.querySelector('.menu-extra').appendChild(this._changelogUi.container);
+      minorActionContainer.append(
+        this._bgFillUi.container,
+        this._copyButtonUi.container,
+      );
+      actionContainer.append(this._downloadButtonUi.container);
+      outputElement.append(this._outputUi.container);
+      this._container.append(this._toastsUi.container, this._dropUi.container);
+      menuExtraElement.append(this._changelogUi.container);
 
       // load previous settings
       this._loadSettings();
 
       // someone managed to hit the preloader, aww
       if (this._preloaderUi.activated) {
-        this._toastsUi.show("Ready now!", {
-          duration: 3000
-        });
+        this._toastsUi.show('Ready now!', { duration: 3000 });
       }
 
       // for testing
       // eslint-disable-next-line no-constant-condition
       if (false) {
         (async () => {
-          this._onInputChange({
-            data: await fetch('test-svgs/car-lite.svg').then(r => r.text()),
```

Comment:
I'm not against this change, but why? I thought it was to avoid putting await within another expression, but another change `(await toast.answer)` does exactly that.